lib/index.js

```diff
@@ -179,6 +200,30 @@ FSTree.defaultIsEqual = function defaultIsEqual(entryA, entryB) {
   return equal;
 };
 
+FSTree.applyPatch = function(input, output, patch, _delegate) {
+  var delegate = assign({}, DEFAULT_DELEGATE, _delegate);
+  for (var i = 0; i < patch.length; i++) {
+    applyOperation(input, output, patch[i], delegate);
+  }
+};
+
+function applyOperation(input, output, operation, delegate) {
+  var method = operation[0];
+  var relativePath = operation[1];
+  var inputPath = path.join(input, relativePath);
+  var outputPath = path.join(output, relativePath);
+
+  if (delegate[method]) {
+    delegate[method](inputPath, outputPath, relativePath);
+  } else {
+    throw new Error('Unknown patch operation method: ' + method);
+  }
+}
+
+function copy(inputPath, outputPath) {
+  symlinkOrCopy.sync(inputPath, outputPath);
+}
```

Comment:
Is there any point to this function now?  should we just inline the `symlinkOrCopy` calls within `DEFAULT_DELEGATE`?